.circleci/config.yml

```diff
@@ -29,12 +29,14 @@ jobs:
       - run:
           name: bring up trellis-ext-db, wait for it to start, run sinopia client integration tests
           command: |
-            docker-compose up -d
-            dockerize -wait http://localhost:8081/healthcheck -timeout 1m
+            docker-compose up -d platform
             cd sinopia_client
+            dockerize -wait http://localhost:8081/healthcheck -timeout 1m
             npm run test
   register_image:
-    <<: *defaults
+    working_directory: ~/sinopia_server
+    docker:
```

Comment:
Don't use defaults because defaults use machine execution context; to interact with docker, we need a docker execution context.